data/json/recipes/recipe_others.json

```diff
@@ -7904,5 +7904,36 @@
     [ [ "wheel", 1 ] ],
     [ [ "motor", 1 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "neoprene",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_OTHER",
+  "skill_used": "cooking",
+  "skills_required": [ "cooking", 4 ],
```

Comment:
"skills_required" isn't needed here. It is meant for specifying secondary skills.